src/bin/common.rs

```diff
@@ -48,455 +273,69 @@ pub struct CliOptions {
   pub benchmark: bool,
   pub threads: usize,
   pub metrics_enabled: MetricsEnabled,
-  pub pass1file_name: Option<OsString>,
-  pub pass2file_name: Option<OsString>,
-  pub save_config: Option<String>,
+  pub pass1file_name: Option<PathBuf>,
+  pub pass2file_name: Option<PathBuf>,
+  pub save_config: Option<PathBuf>,
+  #[cfg(feature = "unstable")]
   pub slots: usize,
   #[cfg(feature = "unstable")]
   pub generate_grain_strength: u8,
 }
 
-#[cfg(feature = "serialize")]
-fn build_speed_long_help() -> String {
```

Comment:
This can be used from the derived parser why did you remove it?

Reply:
How can I use it from the derived parser? I'm probably missing something, I've just been using static strings in doc comments.